### PR TITLE
create test for cart-permission-groups

### PR DESCRIPTION
**Pull request by m-alama**
## PR Description

Create tests for cart-permission-groups
this test `Get_cart_permission_groups_by_cart_id` is included with following the scheme in swagger
but it's introduced a bug https://spryker.atlassian.net/browse/CC-16527
after bug is resolved I will uncomment it

### CODE REVIEW

File: tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot [hunk 1]
```diff
@@ -0,0 +1,66 @@
+*** Settings ***
+Suite Setup    SuiteSetup
+Test Setup    TestSetup
+Resource    ../../../../../../resources/common/common_api.robot
+Default Tags    glue
+
+*** Test Cases ***
+
+# Get_cart_permission_groups_by_cart_id
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot:9]: why is this test commented? if there is a bug this bug should be added to the test as a comment

File: tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    And Each Array Element Of Array In Response Should Contain Nested Property:    [data]    [attributes]    name
+    And Each Array Element Of Array In Response Should Contain Nested Property:    [data]    [attributes]    isDefault
+    And Each Array Element Of Array In Response Should Contain Nested Property:    [data]    [links]    self
+    And Response Body Has Correct Self Link
+
+
+Get_cart_permission_groups_by_id
+    [Setup]    Run Keywords    I get access token for the customer:    ${yves_user_email}
+    ...  AND    I set Headers:    Authorization=${token}
+    When I send a GET request:    /cart-permission-groups/1
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot:58]: move permission group id to a varialbe

File: tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    Then Response status code should be:    200
+    And Response reason should be:    OK
+    And Response Body Parameter Should Be:    [data][type]    cart-permission-groups
+    And Response Body Parameter should Not Be EMPTY:   [data][type]
+    And Response Body Parameter should Not Be EMPTY:   [data][id]
+    And Response Body Parameter should Not Be EMPTY:   [data][attributes][name]
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot:64]: you can also add name to a variable and check against that variable

File: tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    [Setup]    Run Keywords    I get access token for the customer:    ${yves_user_email}
+    ...  AND    I set Headers:    Authorization=${token}
+    When I send a GET request:    /cart-permission-groups
+    Then Response status code should be:    200
+    And Response reason should be:    OK
+    And Each Array Element Of Array In Response Should Contain Property With Value:    [data]    type    cart-permission-groups
+    And Each Array Element Of Array In Response Should Contain Property:    [data]    type
+    And Each Array Element Of Array In Response Should Contain Property:    [data]    id
+    And Each Array Element Of Array In Response Should Contain Nested Property:    [data]    [attributes]    name
+    And Each Array Element Of Array In Response Should Contain Nested Property:    [data]    [attributes]    isDefault
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot:50]: this field is more effectively checked with this keyword:
Each array element of nested array should contain property with value NOT in: [data] [attributes][isDefault] True False

File: tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+Get_cart_permission_groups_by_id
+    [Setup]    Run Keywords    I get access token for the customer:    ${yves_user_email}
+    ...  AND    I set Headers:    Authorization=${token}
+    When I send a GET request:    /cart-permission-groups/1
+    Then Response status code should be:    200
+    And Response reason should be:    OK
+    And Response Body Parameter Should Be:    [data][type]    cart-permission-groups
+    And Response Body Parameter should Not Be EMPTY:   [data][type]
+    And Response Body Parameter should Not Be EMPTY:   [data][id]
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot:63]: check against group ID variable here

Comment (m-alama) [reply tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot:66]: Done

File: tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot (hunk 4 above)
Comment (m-alama) [reply tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot:51]: Done

File: tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot (hunk 3 above)
Comment (m-alama) [reply tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot:67]: Done

File: tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot (hunk 2 above)
Comment (m-alama) [reply tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot]: Done

File: tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot (hunk 1 above)
Comment (m-alama) [reply tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot:10]: I just don't want to write something that is'n really **code** for that I have been mention it in PR description

Comment (OKrivtsova) [reply tests/api/b2b/glue/cart_endpoints/cart-permssion-groups/positive.robot:10]: after the brach is committed and deleted no one will know there is a bug - we add bug refereces to tests that fail or that are commeted.
i your case the test should NOT be commented - only tests that cover untestable/unintegrated features should be commented - please uncommet ad add a bug link

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**